R/gm-auth.R

```diff
@@ -0,0 +1,207 @@
+# This file is the interface between gmailr and the
+# auth functionality in gargle.
+.auth <- gargle::init_AuthState(
+  package     = "gmailr",
+  auth_active = TRUE # TODO: what is this?
+)
+
+# The roxygen comments for these functions are mostly generated from data
+# in this list and template text maintained in gargle.
+gargle_lookup_table <- list(
+  PACKAGE     = "gmailr",
+  YOUR_STUFF  = "your Gmail projects",
+  PRODUCT     = "Google Gmail",
+  API         = "Gmail API",
+  PREFIX      = "gm",
+  AUTH_CONFIG_SOURCE = "gmailr" #TODO: what is this?
```

Comment:
Is this the correct setting for this? Is there any additional setup I need to do to ensure the configuration is correct?

Reply:
If the docs are being built without complaint and "read" OK, then all is good.

The `AUTH_CONFIG_SOURCE` line can be deleted.